benchmark/fft/fft_benchmark.cc

```diff
@@ -188,6 +188,7 @@ int RealMain(int argc, char** argv) {
 
   FFTConfig config;
   FFTConfig::Options options;
+  options.include_check_results = true;
```

Comment:
Please remove this change.

Reply:
It was removed after "set `include_check_results` to true by default" commit. Do you mean I need to remove earlier than this commit?

Reply:
Yeah please remove the from [026674d](https://github.com/kroma-network/tachyon/pull/522/commits/026674d5f604ccf28bded1ad76cf4c8b2b51c4dc).